src/util/world_model.hpp

```diff
@@ -15,26 +15,21 @@ namespace big::world_model
 
 	inline Object spawn(Hash hash, Vector3 location = Vector3(), bool is_networked = true)
 	{
-		STREAMING::REQUEST_MODEL(hash);
-		for (int i = 0; i < 100 && !STREAMING::HAS_MODEL_LOADED(hash); i++)
+		if (entity::request_model(hash))
 		{
-			script::get_current()->yield();
-		}
-		if (!STREAMING::HAS_MODEL_LOADED(hash))
-		{
-			LOG(WARNING) << "Failed to load model " << HEX_TO_UPPER(hash);
-			return 0;
-		}
+			std::call_once(once_flag, setup_backup);
+			memset(g_pointers->m_gta.m_world_model_spawn_bypass, 0x90, patch_size);
 
-		std::call_once(once_flag, setup_backup);
-		memset(g_pointers->m_gta.m_world_model_spawn_bypass, 0x90, patch_size);
+			const auto object = OBJECT::CREATE_OBJECT(hash, location.x, location.y, location.z, is_networked, false, false);
 
-		const auto object = OBJECT::CREATE_OBJECT(hash, location.x, location.y, location.z, is_networked, false, false);
+			memcpy(g_pointers->m_gta.m_world_model_spawn_bypass, backup.data(), patch_size);
```

Comment:
I know this isn't your fault but could you update the patching of the `m_world_model_spawn_bypass` and make it into a `memory::byte_patch`?

Similarly to how it was implemented here:
https://github.com/YimMenu/YimMenu/blob/7efbee2e85ba2aee773e398cd9e66d54e640aca5/src/byte_patch_manager.cpp#L16

And here's an example usage:
https://github.com/YimMenu/YimMenu/blob/7efbee2e85ba2aee773e398cd9e66d54e640aca5/src/backend/looped/self/police.cpp#L19